api-test/fetch-user.js

```diff
@@ -0,0 +1,27 @@
+
+const cardContainer = document.getElementById('card-container');
+const fetchData = async()=>{
+  try{
+    const res = await fetch('https://jsonplaceholder.typicode.com/users')
+    const userData= await res.json();
+    console.log(userData);
+    userData.map((data)=>{
+      const cardDiv=document.createElement('div')
+      cardDiv.setAttribute('class','card');
+      const p1Tag = document.createElement('p')
+      p1Tag.innerText=`UserName : ${data.name}`
+      const p2Tag=document.createElement('p');
+      p2Tag.innerText=`EmailId : ${data.email}`;
+      cardDiv.appendChild(p1Tag);
+      cardDiv.appendChild(p2Tag);
+      cardContainer.appendChild(cardDiv)
+      console.log(data.id,data.name,data.username,data.email)
+    })
+   
+    
```

Comment:
Why so much space here? use prettier

Reply:
okay sir, I'm removing the white spaces, I've used prettier but it's left by mistaken.

Reply:
Sir I've removed the white spaces.